### PR TITLE
python-torch: Bump to 2.0.1 and enable Vulkan support

### PR DESCRIPTION
**Pull request by nviet**
Bump the package to latest version (v2.0.1) and add support for Vulkan API also.

### CODE REVIEW

File: packages/python-torch/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://pytorch.org/
 TERMUX_PKG_DESCRIPTION="Tensors and Dynamic neural networks in Python"
 TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.0.0
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=2.0.1
+TERMUX_PKG_REVISION=4
```
Comment (xtkoba) [inline packages/python-torch/build.sh:6]: Reset (remove) `TERMUX_PKG_REVISION` when bumping `TERMUX_PKG_VERSION`.

**Review by xtkoba — COMMENTED** [review]
I have no idea if this works. Can anyone in the video team confirm this?

**Review by xtkoba — COMMENTED** [review]
Please squash all the commits into one.

**Comment by nviet** [comment]
I'm not able to squash the commits into one. Should I close this PR and reopen a new one with 1 commit only?

File: packages/python-torch/build.sh (hunk 1 above)
Comment (xtkoba) [inline packages/python-torch/build.sh:6]: Reset revision when bumping version.

**Review by xtkoba — COMMENTED** [review]
Will be done on our side instead then. No need to submit a new PR. Thanks.

**Comment by truboxl** [comment]
99% of the time packages do not need vulkan-tools, will fix the vulkan deps either by me or @licy183 

Will this also have the same issue as #16979? I dont want to be blamed for that again...

**Comment by xtkoba** [comment]
@truboxl You are not to blame for protobuf symbol conflict. That should be handled in another PR, if needed, because `python-torch` already depends on both `ffmpeg` and `libprotobuf`.

**Comment by licy183** [comment]
It seems to work fine with tensor operations. But I don't know how to test vulkan with pytorch...

**Comment by xtkoba** [comment]
Mergeable if not causing regressions. Thanks.

**Pull request merged**